Projects/NUCLEO-L432KC/Gate/lib/Gate/cmd.c

```diff
@@ -24,9 +24,7 @@ static void next_json(void)
 
 void check_json(uint16_t carac_nbr)
 {
-    if ((current_table > (JSON_BUF_NUM - 1)) ||
-        (current_table < 0) ||
-        (carac_nbr > (JSON_BUFF_SIZE - 1)))
+    if ((current_table > (JSON_BUF_NUM - 1)) || (current_table < 0) || (carac_nbr > (JSON_BUFF_SIZE - 1)))
```

Comment:
Perhaps we should put OR operator || in the second line.  So, after clang format processing, the conditions should be on different lines.